infrastructure/src/main/java/org/corfudb/infrastructure/BaseServer.java

```diff
@@ -129,14 +168,18 @@ public synchronized void handleMessageSetEpoch(@NonNull CorfuPayloadMsg<Long> ms
      * which monitors the exit code of Corfu. If the exit code is 100, then it resets
      * the server and DELETES ALL EXISTING DATA.
      *
-     * @param msg   The incoming message
-     * @param ctx   The channel context
-     * @param r     The server router.
+     * @param req The incoming request message.
+     * @param ctx The channel context.
+     * @param r The server router.
      */
-    @ServerHandler(type = CorfuMsgType.RESET)
-    private void doReset(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
-        log.warn("Remote reset requested from client {}", msg.getClientID());
-        r.sendResponse(ctx, msg, CorfuMsgType.ACK.msg());
+    @RequestHandler(type = RequestPayloadMsg.PayloadCase.RESET_REQUEST)
```

Comment:
`RESET` is often confused with `RESTART` maybe we can call it something else like `FLASH`, or another descriptive verb that indicates that the state will be nuked.